src/components/Footer/index.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { P } from '@/styles/text';
+import { IconType } from '@/lib/icons';
+import { Flex } from '@/styles/containers';
+import FooterImage from '../../../public/images/footer-image.webp';
+import * as Styles from './styles';
+import Icon from '../Icon';
+
+type SocialLink = {
+  href: string;
+  type: IconType;
+};
+
+const socialLinks: SocialLink[] = [
+  {
+    href: 'https://www.facebook.com/ImmigrationJusticeProjectOfSanDiego/',
+    type: 'facebook',
+  },
+  { href: 'https://twitter.com/ijpsandiego?lang=en', type: 'x' },
+  {
+    href: 'https://www.linkedin.com/company/immigration-justice-project/',
+    type: 'linkedin',
+  },
+  { href: 'https://www.instagram.com/ijp.sandiego/', type: 'instagram' },
+];
```

Comment:
can we refactor the social media links to be in `CONFIG` located in `src/lib/config.ts`?

reasoning: social media links are prone to external changes, so it would be good if we make it easily changeable by the future dev team

i.e.
```typescript
const CONFIG = {
  ...
  facebook: 'https://www.facebook.com/ImmigrationJusticeProjectOfSanDiego/',
  xTwitter: 'https://twitter.com/ijpsandiego?lang=en',
  ...
};
```